### ISSUE TITLE
Can't install Starter Kits currently

### ISSUE THREAD

**Issue by robdekort** [statamic/cli#70]
I know we've seen this issue before, but I can't remember what it was then. Currently we can't seem to install any starter kit using the CLI or using `pls starter-kit:install`:

```
In NewCommand.php line 555: There was a problem installing Statamic with the chosen starter kit!
```

**Comment by robdekort** [comment]
Oh darn, I missed that one! Sorry about this.

**Comment by duncanmcclean** [comment]
Can you provide the full command you're trying to run?

**Comment by robdekort** [comment]
`statamic new site` -> pick any kit
`statamic new site vendor/kit`
`statamic new && php please starter-kit:install vendor/kit`

**Comment by duncanmcclean** [comment]
Do you have anything in your `~/.composer/config.json` pointing to local starter kit repositories?

**Comment by duncanmcclean** [comment]
I'll take a look 👍

**Comment by robdekort** [comment]
Yeah best try installing one I guess, fairly positive it should crap out for you as well.
